ArticleTemplates/assets/scss/modules/_tags.scss

```diff
@@ -10,9 +10,9 @@ Display which keywords an article has been tagged with.
 <div class="tags">
```

Comment:
The first 'Tags' list item can be removed from this comment and the class 'tags__list-item' should be added to the other list items in the comment too.